src/main/kotlin/com/team4099/robot2022/commands/shooter/SpinUpCommand.kt

```diff
@@ -0,0 +1,23 @@
+package com.team4099.robot2022.commands.shooter
+
+import com.team4099.lib.units.derived.rotations
+import com.team4099.lib.units.inRotationsPerMinute
+import com.team4099.lib.units.perMinute
+import com.team4099.robot2022.config.Constants.ShooterConstants
+import com.team4099.robot2022.subsystems.Shooter
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class SpinUpCommand(val target: Double = ShooterConstants.ShooterState.SPIN_UP.rotationsPerMinute) :
+  CommandBase() {
+  init {
+    addRequirements(Shooter)
+  }
+
+  override fun initialize() {
+    Shooter.setVelocity(target)
+  }
+
+  override fun isFinished(): Boolean {
+    return ShooterConstants.TARGET_VELOCITY - Shooter.shooterVelocity <= ShooterConstants.TARGET_VELOCITY_THRESHOLD
```

Comment:
what happens here if the shooter is spinning up to the right speed but is oscillating just past the target velocity (so that it crosses the threshold but is not going to stay there for now)?

also, the check for a denoised shooter velocity or whatever should probably go in the shooter (so that you can just return `Shooter.atTargetVelocity` or something